lib/mayaUsd/ufe/CMakeLists.txt

```diff
@@ -274,12 +231,13 @@ if(CMAKE_UFE_V4_FEATURES_AVAILABLE)
         UsdConnectionHandler.h
         UsdUINodeGraphNode.h
         UsdUINodeGraphNodeHandler.h
-        UsdTransform3dRead.h
         UsdUndoConnectionCommands.h
         UsdUndoCreateStageWithNewLayerCommand.h
     )
 endif()
 
+add_subdirectory(trf)
```

Comment:
PierreB had a previous PR to move all the transform3d code (in maya-usd) into a sub-folder called `trf` - to make it easier to find all the transform3d code (it would all be together). So I kept the same folder name as he used.